gammapy/cube/core.py

```diff
@@ -526,6 +526,14 @@ def to_fits(self):
 
         return hdu_list
 
+
+    def to_image_list(self):
+        """ Writes SkyCube to `gammapy.image.SkyImageList` as a list of `gammapy.image.SkyMap` objects.
+        """
+        skymaps = [SkyMap(self.name, Quantity(slice_data, self.data.unit), self.wcs) for slice_data in self.data]
```

Comment:
Actually I now see that the `SkyCube.wcs` is already the 2-dimensional celestial WCS slice ... this is done in `SkyCube.read`:
https://github.com/gammapy/gammapy/blob/master/gammapy/cube/core.py#L158

I don't know if I coded that up a year or two ago or if someone changed it to be like this in the meantime.

In any case the class docstring is correct ... it should say that the `wcs` member is the 2-dim celestial WCS here:
https://github.com/gammapy/gammapy/blob/master/gammapy/cube/core.py#L45

@OlgaVorokh So you can just pass the cube WCS to the image WCS, and don't have to do any WCS computation or axis order check. Could you please update the cube docstring to a correct one-sentence statement what `WCS` is in this PR?